x-pack/plugins/ml/public/application/management/jobs_list/components/jobs_list_page/jobs_list_page.tsx

```diff
@@ -108,18 +111,26 @@ export const JobsListPage: FC<{
   coreStart: CoreStart;
   share: SharePluginStart;
   history: ManagementAppMountParams['history'];
-}> = ({ coreStart, share, history }) => {
+  spaces?: SpacesPluginStart;
+}> = ({ coreStart, share, history, spaces }) => {
+  const spacesEnabled = spaces !== undefined;
   const [initialized, setInitialized] = useState(false);
   const [accessDenied, setAccessDenied] = useState(false);
+  const [showRepairFlyout, setShowRepairFlyout] = useState(false);
   const [isMlEnabledInSpace, setIsMlEnabledInSpace] = useState(false);
-  const tabs = useTabs(isMlEnabledInSpace);
+  const tabs = useTabs(isMlEnabledInSpace, spacesEnabled);
   const [currentTabId, setCurrentTabId] = useState(tabs[0].id);
   const I18nContext = coreStart.i18n.Context;
+  const spacesContext = useMemo(() => createSpacesContext(coreStart.http), []);
```

Comment:
I tested 21a176c locally with Spaces disabled, and I still noticed that when you view the ML Jobs page, a call is being made to `/internal/spaces/_active_space` which results in a 404 error.

This line appears to be the culprit -- it creates a `SpacesManager` object, which attempts to check the active space in its constructor. I guess you'll need to pass `spacesEnabled` into this function and avoid creating a `SpacesManager` if it's false.

Reply:
not sure how I missed this one?!
Fixed in https://github.com/elastic/kibana/pull/83320/commits/e10663d216ba34ec4f56e141e3c22890014c9e9d
Thanks for checking it.